gemhardware/devices/include/gem/hw/devices/amc/HwGenericAMC.h

```diff
@@ -4,774 +4,702 @@
 #define GEM_HW_HWGENERICAMC_H
 
 #include "gem/hw/devices/GEMHwDevice.h"
+#include "ctp7_modules/common/amc.h"
+#include "ctp7_modules/common/amc/daq.h"
+#include "ctp7_modules/common/amc/ttc.h"
```

Comment:
are these includes needed here? or can they be moved to the implementation?

Reply:
done